src/main/java/randoop/main/Minimize.java

```diff
@@ -1109,11 +1113,7 @@ public String call() throws Exception {
       }
       bufReader.close();
     } catch (IOException e) {
-      // Error with readLine() or close().
-      System.err.println("I/O error reading line from trace.");
-      e.printStackTrace();
-      System.exit(1);
-      return null;
+      throw new RuntimeException("Buffered reader failed: " + e.getMessage());
```

Comment:
You will preserve more information by using a chained exception, as in `new RuntimeException(e)`.